### ISSUE TITLE
bug: Typography font selection dropdown not set to first result

### ISSUE THREAD

**Issue by anditosan** [uxbox/uxbox#5272]
### Steps To Reproduce

Access Assets > Typography > Click + > Click font selection dropdown > Search for a font

Dropdown shows second font as the result leading the user to have to scroll up to find the appropriate font. I tested this using Inter font uploaded from the internet while keeping the provided inter font in PenPot.

### Expected behavior

Dropdown results should be set to the first item to show.

### Actual behavior

Showing second item as the first result.

### Screenshots or video

<img width="458" alt="SCR-20241109-osad" src="https://github.com/user-attachments/assets/146fdc0d-a4dc-40f3-8b86-303d576db44f">

<img width="469" alt="SCR-20241109-osbr" src="https://github.com/user-attachments/assets/e7a9a9d8-27ee-4421-9057-438293cdfbca">

### Desktop (please complete the following information)

Web

**Comment by madalenapmelo-kp** [comment]
Hi @anditosan,

Could you share a screen recording of this behavior?

**Comment by anditosan** [comment]
> Hi @anditosan,
> 
> Could you share a screen recording of this behavior?

Yes, basically, whenever I start adding a new typography token, the system defaults to the system's default font and not the last selected font, in our case, Inter.

**Comment by madalenapmelo-kp** [comment]
Hi @anditosan,

Thanks for reporting this! We've added this to our backlog on Taiga so that we can look further into it, you can find the details here: https://tree.taiga.io/project/penpot/issue/9502

**Comment by Alotor** [comment]
I think this issue has been solved. If you find the bug is still happen reopen the issue. Thanks
